### DISCUSSION TITLE
First login not possible

### DISCUSSION THREAD

**Discussion by funked00** [Budibase/budibase#2407] [Help] [answered]
Hi,

after creating the admin user I cannot login. For a short time a green "Logged in successfuly" banner appears but then I'm always redirected to the login page again. I tried different Browsers.

Any advise where to start debugging?

Best regards
Daniel

**Comment by kevmodrome** [comment, marked as answer]
For next time please include more information about your browser and how you're running Budibase.

Have a look at this issue: https://github.com/Budibase/budibase/discussions/2375
